tests/neg-macros/newClassExtendsNoParents.check

```diff
@@ -0,0 +1,19 @@
+
+-- Error: tests/neg-macros/newClassExtendsNoParents/Test_2.scala:1:25 --------------------------------------------------
+1 |def test: Any = makeClass("foo") // error
+  |                ^^^^^^^^^^^^^^^^
+  |                Exception occurred while executing macro expansion.
+  |                java.lang.AssertionError: assertion failed: First parent must be a class
+  |                	at scala.runtime.Scala3RunTime$.assertFailed(Scala3RunTime.scala:8)
+  |                	at scala.quoted.runtime.impl.QuotesImpl$reflect$Symbol$.newClass(QuotesImpl.scala:2464)
```

Comment:
This checkfile is too specific: it would need to be updated every time we add/remove a line in QuotesImpl.scala (hence why the tests are failing right now).